src/transformers/integrations/peft.py

```diff
@@ -432,7 +432,8 @@ def active_adapters(self) -> List[str]:
 
     def active_adapter(self) -> str:
         warnings.warn(
-            "The `active_adapter` method is deprecated and will be removed in a future version.", FutureWarning
+            "The `active_adapter` method is deprecated and will be removed in a future version.",
```

Comment:
I think what happened here and further below is that ruff did not take into account the repo specific settings of 119 chars per line from the `pyproject.toml`. Instead, it uses the default 88 chars per line.

Reply:
Not sure if it worked properly, as there are still a bunch of changes in this PR that break lines unrelated to you changes.